### PR TITLE
feat: LT-948 Show assigned users on the case screen

### PR DESCRIPTION
**Pull request by pkonopacki**
- added section to show assigned users on the case screen
- e2e test added

### CODE REVIEW

File: docker.env [hunk 1]
```diff
@@ -1,24 +0,0 @@
-DEBUG=True
```
Comment (jfaracik) [inline docker.env]: This file shouldn't have been deleted! :)

Comment (pkonopacki) [reply docker.env]: Restored Sir

**Comment by andytaylorkainos** [comment]
Howcome you had to update assets/shared?

**Comment by BenAndrew-Kainos** [comment]
Why has so much content changed (have you made sure to update your submodules to what's in master?)

File: templates/layouts/case.html [hunk 2]
```diff
@@ -71,6 +71,31 @@ <h1 class="govuk-heading-l govuk-!-margin-0">
 		<p class="govuk-caption-m">{% lcs 'cases.ApplicationPage.NO_CASE_OFFICER' %}</p>
 	{% endif %}
 
+	<div class="govuk-!-margin-bottom-6" id="case_queues">
+		{% for queue_name in case.queue_names %}
+			<div class="govuk-caption-m">
+				{{ queue_name }}
+			</div>
+		{% empty %}
```
Comment (BenAndrew-Kainos) [inline templates/layouts/case.html]: This is nice :)

File: templates/layouts/case.html [hunk 3]
```diff
@@ -71,6 +71,31 @@ <h1 class="govuk-heading-l govuk-!-margin-0">
 		<p class="govuk-caption-m">{% lcs 'cases.ApplicationPage.NO_CASE_OFFICER' %}</p>
 	{% endif %}
 
+	<div class="govuk-!-margin-bottom-6" id="case_queues">
+		{% for queue_name in case.queue_names %}
+			<div class="govuk-caption-m">
+				{{ queue_name }}
```
Comment (BenAndrew-Kainos) [inline templates/layouts/case.html]: Could you put it on one line like below?

File: templates/layouts/case.html [hunk 4]
```diff
@@ -71,6 +71,31 @@ <h1 class="govuk-heading-l govuk-!-margin-0">
...
+			</div>
+		{% empty %}
+			<span class="govuk-caption-m">{% lcs 'cases.ApplicationPage.NO_QUEUES_ASSIGNED' %}</span>
+		{% endfor %}
+	</div>
+
+	<div class="govuk-!-margin-bottom-6" id="assigned_users">
+		{% for user in case.users %}
+			<div class="govuk-caption-m">
+				<span style="opacity: 0.6;">{{ user.queue }}:</span>
```
Comment (BenAndrew-Kainos) [inline templates/layouts/case.html]: Don't think we should be setting inline styles or changing opacity. Please use an existing styling (@jfaracik is the man to ask about this)

File: ui_automation_tests/features/view_case_details.feature [hunk 5]
```diff
@@ -25,3 +25,16 @@ Feature: I want to view the case details of a case
...
+    Given I create application or application has been previously created
+    And I sign in to SSO or am signed into SSO
+    And I am an assigned user for the case
+    When I go to application previously created
+    Then I see an end user
+    And I see an ultimate end user
+    And I see a third party
+    And I see a consignee
+    And I see assigned queues
+    And I see assigned users
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/features/view_case_details.feature]: newline at end of file

File: ui_automation_tests/features/view_case_details.feature [hunk 6]
```diff
@@ -25,3 +25,16 @@ Feature: I want to view the case details of a case
...
     When I go to application previously created
     Then I see that changes have been made to the case
+
+    @LT_948_can_see_assigned_users @regression
+    Scenario: Gov user can see which users are assigned to a case from the case screen
+    Given I create application or application has been previously created
+    And I sign in to SSO or am signed into SSO
+    And I am an assigned user for the case
+    When I go to application previously created
+    Then I see an end user
```
Comment (BenAndrew-Kainos) [inline ui_automation_tests/features/view_case_details.feature]: Do we not already test seeing parties in another e2e? (If we don't then keep this and add 'see_parties' to the name)

**Pull request merged**